---
title: "Postdoctoral Positions"
date: 2021-07-13T15:34:30-04:00
categories:
tags:
---

<h2>Project 1</h2>
<body>
Job Title: Postdoctoral fellow<br />
Grant end date :  31/07/2024<br />
Contract Period : 01/08/2021 â€“ 31/07/2024 (3 years)<br /><br />

The   project   aims   to   develop   the   AI   method   to   detect   present   microbes   in   a sample   (ie.   throat-swab,   urine,   blood)   sequenced   using nanopore   technology. Nanopore sequencers read DNA/RNA fragments while they are passing through a tiny pore. Passage of nucleotides through pore results in a signal which is in a downstream process converted to the sequence of nucleotides.<br /><br />
The project will start with recognising each microbe using a pattern matching approach, like those for rapid song recognition. These methods will be performed on either raw signal or basecalled sequence of nucleotides. We aim to use deep learning   methods   to   find   a   concise   representation   of   each   known   microbe genome.<br /><br />
The fast and accurate detection of microbes that cause infections disease can reduce unnecessary usage of antibiotics, speed up recovery and sometimes even save lives. When clinicians do not have a clue about the possible pathogen, the current methods can last days to identify the pathogen correctly.<br /><br />
In this role, one will be part of a multi-disciplinary team consisting of computer scientists, biologists, and bioinformaticians. Duties include developing algorithms and AI methods for the fast identification of present microbes.<br /><br />

Requirements:<br />
- Phd in Computer Science, Bioinformatics, or a related discipline.<br />
- Strong programming skills (ie. C++, Rust, Python)<br />
- Active knowledge of PyTorch, Tensorflow or similar deep learning frameworks<br />
- Excellent communication skills in written and spoken English<br />

<body/>
  <br />
  
  
<h2>Project 2</h2>
<body>
Job Title: Postdoctoral fellow<br /><br />

The project aims to develop a new method for T2T haplotype resolved de novo assembly. To achieve this goal, we plan to use algorithms and AI methods ongraphs   and   strings.   The   project   will   start   with   our   recently   developed   Raven assembler <a href="https://github.com/lbcb-sci/raven">https://github.com/lbcb-sci/raven.</a><br/>
  <br/>
The recent successful reconstruction of haploid Chm13 human cell line proved that it is possible to reconstruct a genome from telomere to telomere correctly using existing sequencing technologies.<br/>
  <br/>
This project is part of an ambitious Asian Reference Genome project that aims to correctly   reconstruct   genomes   of   150   participants   (Chinese,   Malay   and   India), integrate   them   into   a   pangenome,   and   create   tools   to   use   them   in   the Singaporean National Precision Medicine program.<br />
  <br />
In this role, one will be part of a multi-disciplinary team consisting of computer scientists, biologists, and bioinformaticians. Duties include primarily developing algorithms on graphs and strings for de novo genome assembly. Still, the fellow will also be encouraged to join the team in the development of AI methods.<br /><br />

Requirements:<br />
- Phd in Computer Science, Bioinformatics, or a related discipline.<br />
- Strong programming skills (ie. C++, Rust, Python)<br />
- Deep knowledge of Algorithms and Data Structures<br />
- Excellent communication skills in written and spoken English<br />
<br />
 Optional:<br />
- Knowledge of modern supervised, unsupervised or reinforcement learning methods<br />
- Active knowledge of PyTorch, Tensorflow or similar deep learning frameworks<br />
<body/>
  <br />

  
<h2>Project 3</h2>
<body>
Job Title: Postdoctoral fellow<br /><br />

The project aims to develop a new generation of mRNA based vaccines. The goal of the computational part of the project is to develop methods for predicting RNA tertiary structures and optimizing RNA stability.<br />
  <br />
The   recent   breakthrough   of   AlphaFold2   in   accurately   predicting   protein structures   paves   the   way   for   applying   deep   learning   techniques   to   grand challenges in biology. However, it is critical to emphasize that learning RNA tertiary structures is significantly more challenging than protein structures. Only 15,000 chains in 5,000 RNA structures are present in the PDB, 30-fold less than the 150,000 training proteins. The lack of experimental training data requires us to: (1) augment the training set with new experimental data, (2) design more theoretically expressive learning techniques than AlphaFold2, which rely on a standard   supervised   learning   framework   and   a   classical   Transformer-based architecture.<br />
  <br />
In this role, one will be part of a multi-disciplinary team consisting of computer scientists, biologists, and bioinformaticians. Duties include primarily developing new AI methods for RNA structure predictions.<br /><br />

Requirements:<br />
- PhD in Computer Science, Bioinformatics, or a related discipline.<br />
- Strong programming skills (ie. C++, Rust, Python)<br />
- Knowledge of modern supervised, unsupervised or reinforcement learning methods<br />
- Active knowledge of PyTorch, Tensorflow or similar deep learning frameworks<br />
- Excellent communication skills in written and spoken English<br />
<body/>
